IS7024Project/Pages/ParkList.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@page
```

Comment:
New page created to display your list of parks. I also changed the link to the privacy page to this new page in your layout page for better navigation.